infra/gcp/terraform/k8s-infra-prow-build/prow-build/resources/monitoring/grafana/datasources.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: datasources
+  namespace: monitoring
+data:
+  datasources.yaml: |
+    apiVersion: 1
+    datasources:
+    - access: proxy
+      isDefault: true
+      name: Prometheus Main
+      type: prometheus
+      url: http://prometheus-operated.monitoring:9090
+    - access: proxy
+      isDefault: false
+      name: Loki
+      type: loki
+      url: http://loki-stack.monitoring:3100
```

Comment:
I don't see that we have Loki running in this cluster, please correct me if I'm wrong.

Reply:
right, I also did not include Loki as the logs are already shipping to GCP logging. Removing...